react/package.json

```diff
@@ -15,6 +17,7 @@
     "react-onesignal": "^2.0.4",
     "react-phone-number-input": "^3.4.12",
     "react-toastify": "^11.0.5",
+    "topojson-client": "^3.1.0",
```

Comment:
Same comment as above, it appears we pull topojson js from the d3js domain on the page itself so we should not bundle it here as well.